### ISSUE TITLE
Gettings Error while Exporting Firestore

### ISSUE THREAD

**Issue by jefrydco** [jloosli/node-firestore-import-export#25]
### Expected behavior
Previously I use this library is working well, then when I use this is again it display an error like this:
```
The behavior for Date objects stored in Firestore is going to change
AND YOUR APP MAY BREAK.
To hide this warning and ensure your app does not break, you need to add the
following code to your app before calling any other Cloud Firestore methods:

  const firestore = new Firestore();
  const settings = {/* your settings... */ timestampsInSnapshots: true};
  firestore.settings(settings);

With this change, timestamps stored in Cloud Firestore will be read back as
Firebase Timestamp objects instead of as system Date objects. So you will also
need to update code expecting a Date to instead expect a Timestamp. For example:

  // Old:
  const date = snapshot.get('created_at');
  // New:
  const timestamp = snapshot.get('created_at');
  const date = timestamp.toDate();

Please audit all existing usages of Date when you enable the new behavior. In a
future release, the behavior will change to the new behavior, so if you do not
follow these steps, YOUR APP MAY BREAK.
Auth error:Error: invalid_grant
Auth error:Error: invalid_grant
```

The last two line are repeated several times.

### Actual behavior
It doesn't display the error.

### Steps to reproduce the behavior
Run firestore-export command as usual

**Comment by jloosli** [comment]
@jefrydco 

Sorry for the late reply...I've been out of town and traveling, so I'll have to take a closer look into this and get back to you.

**Comment by jloosli** [comment]
Fixed in #29
